images/stackstorm/bin/entrypoint.sh

```diff
@@ -18,6 +18,8 @@ crudini --set ${ST2_CONF} mistral api_url http://127.0.0.1:9101
 crudini --set ${ST2_CONF} mistral v2_base_url http://127.0.0.1:8989/v2
 crudini --set ${ST2_CONF} messaging url \
   amqp://${RABBITMQ_DEFAULT_USER}:${RABBITMQ_DEFAULT_PASS}@${RABBITMQ_HOST}:${RABBITMQ_PORT}
+crudini --set ${ST2_CONF} coordination url \
+  redis://${REDIS_PASSWORD}@${REDIS_HOST}:${REDIS_PORT}
```

Comment:
For some reason, if I add a `:` before the password, then st2api does not start successfully. So I will leave the coordination url as is.